packages/curve-ui-kit/src/shared/routes.ts

```diff
@@ -36,52 +35,55 @@ export const DAO_ROUTES = {
 export const AppNames = ['dex', 'lend', 'crvusd', 'dao'] as const
 export type AppName = (typeof AppNames)[number]
 
-const getAppRoot = (app: AppName) => `/${app}`
+export const AppMenuOptions = ['dex', 'llama', 'dao'] as const // llamalend contains links to crvusd and lend
+export type AppMenuOption = (typeof AppMenuOptions)[number]
 
-export const APP_LINK: Record<AppName, AppRoutes> = {
+export const APP_LINK: Record<AppMenuOption, AppRoutes> = {
   dex: {
-    root: getAppRoot('dex'),
     label: 'DEX',
-    pages: [
-      { route: DEX_ROUTES.PAGE_SWAP, label: () => t`Quickswap` },
-      { route: DEX_ROUTES.PAGE_POOLS, label: () => t`Pools` },
-      { route: DEX_ROUTES.PAGE_CREATE_POOL, label: () => t`Pool Creation` },
-      { route: DEX_ROUTES.PAGE_DASHBOARD, label: () => t`Dashboard` },
+    routes: [
+      { app: 'dex', route: DEX_ROUTES.PAGE_SWAP, label: () => t`Quickswap` },
+      { app: 'dex', route: DEX_ROUTES.PAGE_POOLS, label: () => t`Pools` },
+      { app: 'dex', route: DEX_ROUTES.PAGE_CREATE_POOL, label: () => t`Pool Creation` },
+      { app: 'dex', route: DEX_ROUTES.PAGE_DASHBOARD, label: () => t`Dashboard` },
     ],
   },
-  crvusd: {
-    root: getAppRoot('crvusd'),
-    label: 'crvUSD',
-    pages: [
-      { route: CRVUSD_ROUTES.PAGE_MARKETS, label: () => t`Markets` },
-      ...(isBeta ? [{ route: CRVUSD_ROUTES.BETA_PAGE_MARKETS, label: () => t`Markets (beta)` }] : []),
-      { route: CRVUSD_ROUTES.PAGE_PEGKEEPERS, label: () => t`Peg Keepers` },
-      { route: CRVUSD_ROUTES.PAGE_CRVUSD_STAKING, label: () => t`Savings crvUSD` },
+  llama: {
+    label: 'Llamalend',
+    routes: [
+      { app: 'lend', route: LEND_ROUTES.PAGE_MARKETS, label: () => t`Lend` },
```

Comment:
could maybe combine `LEND_ROUTES` and `CRVUSD_ROUTES` already so it's less confusing

Reply:
IMO that is more confusing as going from one app to the other still incurs some initialization. Also the ROUTES object only has strings, I would have to completely change the format of that to add an `app` field. We cannot prepare the full path beforehand as the network address can change.

Ideally the routes should be callbacks e.g.
```typescript
const ROUTES = {
  dex: {
    pools: ({network}) => `dex/${network}/pools`,
  ...
}
```